core/pta/remote_attestation/cbor.c

```diff
@@ -0,0 +1,190 @@
+#include "cbor.h"
```

Comment:
Missing license and copyright header files.
Ditto for some of the other new header and C source files.

Reply:
DONE